client/packages/invoices/src/Prescriptions/api/hooks/useDiagnosisOptions.ts

```diff
@@ -0,0 +1,32 @@
+import { DiagnosisNode, useQuery } from '@openmsupply-client/common';
+import { usePrescriptionGraphQL } from '../usePrescriptionGraphQL';
+import { DIAGNOSIS, LIST } from './keys';
+
+export const useDiagnosisOptions = () => {
```

Comment:
Went with a specific `Options` hook, less generic but fits the use case. Interested if others like this idea...